ballerina/client.bal

```diff
@@ -13,3 +13,1364 @@
+    # + return - successful 
+    resource isolated function put articles/[int id](update_article_request payload, UpdateArticleHeaders headers = {}) returns article|error {
+        string resourcePath = string `/articles/${getEncodedUri(id)}`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->put(resourcePath, request, httpHeaders);
+    }
+
+    # Update a company
+    #
+    # + id - The unique identifier for the company which is given by Intercom
+    # + headers - Headers to be sent with the request 
+    # + return - Successful 
+    resource isolated function put companies/[string id](UpdateCompanyHeaders headers = {}) returns company|error {
+        string resourcePath = string `/companies/${getEncodedUri(id)}`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        return self.clientEp->put(resourcePath, request, httpHeaders);
+    }
+
+    # Update a contact
+    #
+    # + id - id
+    # + headers - Headers to be sent with the request 
+    # + return - successful 
+    resource isolated function put contacts/[string id](contacts_id_body payload, UpdateContactHeaders headers = {}) returns contact|error {
+        string resourcePath = string `/contacts/${getEncodedUri(id)}`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->put(resourcePath, request, httpHeaders);
+    }
+
+    # Update a conversation
+    #
+    # + id - The id of the conversation to target
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - conversation found 
+    resource isolated function put conversations/[int id](update_conversation_request payload, UpdateConversationHeaders headers = {}, *UpdateConversationQueries queries) returns conversation|error {
+        string resourcePath = string `/conversations/${getEncodedUri(id)}`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->put(resourcePath, request, httpHeaders);
+    }
+
+    # Update a data attribute
+    #
+    # + id - The data attribute id
+    # + headers - Headers to be sent with the request 
+    # + return - Successful 
+    resource isolated function put data_attributes/[int id](update_data_attribute_request payload, UpdateDataAttributeHeaders headers = {}) returns data_attribute|error {
```

Comment:
It seems like the generated record type names are not Ballerina friendly. @lnash94 Can we use the type name sanitation features which we have in openAPI tool since we are using Update10?